### PR TITLE
doc: add basic troubleshooting guide for machines

### PR DESCRIPTION
**Pull request by gibfahn**
Would be great to get some of these Tips & Tricks written down.

Refs: https://github.com/nodejs/build/issues/775#issuecomment-336473604

Reviews would be really helpful, I'm making half of this up as I go along...

#### TODO:
- [ ] https://ci.nodejs.org/job/git-delete-branches

### CODE REVIEW

File: doc/fixing-machines.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+When the machine comes back the Jenkins slave should reconnect automatically
+(check on ci.nodejs.org). If it doesn't see the next step.
+
+### Restart the Jenkins agent
+
+Most machines have a service to restart the Jenkins agent. Try one of:
+
+```bash
+# Systemd init (some newer Linux distros):
+systemctl start jenkins.service
```
Comment (gibfahn) [inline doc/fixing-machines.md]: This is a total guess.

Comment (mhdawson) [reply doc/fixing-machines.md]: I think it is `systemctl jenkins` start and `systemctl jenkins stop`

File: doc/fixing-machines.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+# System V init (older Linux distros):
+/etc/init.d/jenkins start
+# AIX:
+/etc/rc.d/rc2.d/S20jenkins start
+# Things we don't have a service for yet:
+/home/iojs/start.sh
+```
+
```
Comment (mhdawson) [inline doc/fixing-machines.md]: Maybe add a comment that you can figure out which by looking at the ansible files for the machine.

Comment (mhdawson) [reply doc/fixing-machines.md]: I see maybe that is what you were saying in the next line :). Maybe a few more words to explain that (ie what service files are) would make it more obvious.

**Pull request merged**